examples/metrics/prometheus.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020, OpenTelemetry Authors
```

Comment:
I think this example could be integrated in the already existing metrics examples likes we do in tracing: https://github.com/open-telemetry/opentelemetry-python/blob/96943b242fc56b56e65695fc092bcbee7b32b27f/examples/basic_tracer/tracer.py#L27-L36

Having multiple / very similar examples is a maintenance nightmare.

Reply:
Created #423 to track this